### PR TITLE
QAE-309 - E2E test case for gutenberg list block

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

- Create a new post
- insert gutenberg List block and add some content
- Check width

### Screenshots

https://share.getcloudapp.com/Qwu4YNRe

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/block-editor/list-block/width.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/block-editor/list-block/width.test.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	it( 'assert default width of the list in the block editor', async () => {
+		await createNewPost( { postType: 'post', title: 'test List' } );
+		await page.click( '[aria-label="Settings"]' );
+		await insertBlock( 'List' );
+		await page.keyboard.type( 'one' );
+		await page.keyboard.press( 'Enter' );
+		await page.keyboard.type( 'two' );
+		await page.waitForSelector( '.wp-block-list' );
+		await expect( {
+			selector: '.wp-block-list',
```
Comment (GauravKhupse) [inline tests/e2e/specs/block-editor/list-block/width.test.js:12]: Could you please verify selector which is missing in the frontend end:- https://share.bsf.io/xQuqL0Br

Comment (bsfsnehal) [reply tests/e2e/specs/block-editor/list-block/width.test.js:12]: @SimranChougule1 please resolve this

Comment (SimranChougule1) [reply tests/e2e/specs/block-editor/list-block/width.test.js:12]: we are not checking on frontend

**Comment by bsfsnehal** [comment]
Closing this PR due to expected & receive value difference error for width option on local as well as github.

**Pull request closed without merging**